src/screens/LoginScreen/index.jsx

```diff
@@ -0,0 +1,130 @@
+import React, {useState} from 'react';
+import {
+  View,
+  Image,
+  Text,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+} from 'react-native';
+import TextField from '../../components/TextField';
+import ButtonComponent from '../../components/ButtonComponent';
+import color from '../../constant/color';
+import {useNavigation} from '@react-navigation/native';
+import {useDispatch, useSelector} from 'react-redux';
+import {userLogin} from '../../redux/actions/userAction';
+import ImageBtn from '../../components/ImageBtn';
+import Icon from 'react-native-vector-icons/AntDesign';
+import TxtComponent from '../../components/TxtComponent';
+import {styles} from './styles';
+
+const LoginScreen = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [validation, setValidation] = useState({
+    emailError: '',
+    passwordError: '',
+  });
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const loginDetails = useSelector(state => state.reducer.loginDetails);
+
+  const userLoginDetails = () => {
```

Comment:
use a better naming like loginHandler,loginBtnHandler.  This name is a bit ambiguous